sig-architecture/charter.md

```diff
@@ -11,6 +11,7 @@ The SIG Architecture maintains and evolves the design principles of Open Cluster
 - *API definitions*
 - *Architectural renderings*
 - *API conventions*
+- [API changes](api_changes.md)
```

Comment:
API upgrades?

Reply:
I still prefer to `api_changes.md`, because we may add more content into the file later for api changes without version upgrade, so IMO `api_changes.md` would be a more common name.